src/puppeteer.js

```diff
@@ -210,6 +210,18 @@ export const launchPuppeteer = (options = {}) => {
     if (options.useApifyProxy && options.proxyUrl) throw new Error('Cannot combine "options.useApifyProxy" with "options.proxyUrl"!');
 
     const puppeteer = getPuppeteerOrThrow();
+
+    // Add stealth
+    if (options.stealth) {
```

Comment:
I don't think that we need to be overriding `puppeteer.launch()`. This could be just added at the end of this function.

BTW @mnmkng rewrote this function to await/async so after we merge Live view PR it's going to be easier.